src/Machine.js

```diff
@@ -584,6 +614,192 @@ class Machine {
     }
   }
 
+  /**
+   * Create folder if path exists
+   * @param {string} dirPath path to directory
+   * @private
+   */
+  _mkdirSync(dirPath) {
+    try {
+      fs.mkdirSync(dirPath);
+    } catch (err) {
+      // if it is not "Exist error" then log it
+      if (err.code !== 'EEXIST') this.logger.error(err);
+    }
+  }
+
+  /**
+   * Create all missing folders in the current path
+   * @param {string} dirPath path to directory
+   * @private
+   */
+  _mkdirpSync(dirPath) {
```

Comment:
It's a terrible practice to give two function similar names that differ with just one character. Especially if this character is in the middle of the string, especially if this character is lowercase.